### PR TITLE
build: separate pkg-config input file for cmake

### PR DESCRIPTION
**Pull request by recp**

### CODE REVIEW

**Comment by q234rty** [comment]
I can confirm this fixes the autotools build.

**Comment by eli-schwartz** [comment]
I think that #259 should just be reverted -- it breaks both autotools and cmake.

File: cglm.cmake.pc.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix="${prefix}"
+libdir="${exec_prefix}/lib"
+includedir="${prefix}/include"
```
Comment (Artturin) [inline cglm.cmake.pc.in:4]:
```suggestion
prefix=@CMAKE_INSTALL_FULL_PREFIX@
exec_prefix="${prefix}"
libdir=@CMAKE_INSTALL_FULL_LIBDIR@
includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Comment (eli-schwartz) [reply cglm.cmake.pc.in:4]: So does simply reverting the original broken commit, surely?

Comment (Artturin) [reply cglm.cmake.pc.in:4]: https://github.com/recp/cglm/pull/272

**Comment by recp** [comment]
@q234rty, @eli-schwartz, @Artturin I'm going to close this PR without merge, since there is better one ( thanks to @Artturin ): https://github.com/recp/cglm/pull/272

many thanks for your feedbacks, investigations, collaboration

**Pull request closed without merging**